people-ops/time-off-and-overtime.html.md

```diff
@@ -27,6 +17,27 @@ possible.
 Holidays accrued in a year must be used within 18 months: e.g., the remaining days on 31/12/2019
```

Comment:
I think it's better to change `Holidays` for `Vacation days` here. Unless it's related to the new Holidays section described below.

Reply:
You're right @igorbp!